inference-engine/src/transformations/src/transformations/op_conversions/convert_gather_0d.cpp

```diff
@@ -10,17 +10,18 @@
 #include <vector>
 
 #include <ngraph/opsets/opset1.hpp>
+#include <ngraph/opsets/opset7.hpp>
 #include <ngraph/rt_info.hpp>
 #include <ngraph/pattern/op/wrap_type.hpp>
 
 NGRAPH_RTTI_DEFINITION(ngraph::pass::ConvertGather0D, "ConvertGather0D", 0);
 
 ngraph::pass::ConvertGather0D::ConvertGather0D() {
     MATCHER_SCOPE(ConvertGather0D);
-    auto gather = ngraph::pattern::wrap_type<opset1::Gather>();
+    auto gather = ngraph::pattern::wrap_type<opset7::Gather>();
```

Comment:
This transformation is not used in common optimization pipeline. Please revert changes.

Reply:
reverted